ctapipe/io/datawriter.py

```diff
@@ -46,8 +46,9 @@ def _get_tel_index(event, tel_id):
 #   (meaning readers need to update scripts)
 # - increase the minor number if new columns or datasets are added
 # - increase the patch number if there is a small bugfix to the model.
-DATA_MODEL_VERSION = "v2.2.0"
+DATA_MODEL_VERSION = "v3.0.0"
```

Comment:
I think because it renamed an existing field, so that would break reading previously written data. Probably we should update the version just before release in any case, rather than using values in each PR.

Reply:
Ah, right, `core_uncert` to `core_uncert_x` / `core_uncert_y`.

So this is not only about the tilted / non-tilted frame?

Reply:
Hi Max,
this specific change it's because when I was working on this, I noticed that the other reconstructors (intersection and Impact) were recording `core_x` and `core_y` (in the ground frame), but computing and recording the error on the 2D quantity which is inconsistent.
Either we record `core` and `core_uncer`, or (as I chose to do) we record `core_x`, `core_uncert_x`, `core_y`, `core_uncert_y` and then subsequent analysis steps or benchmarks can use the 1D quantities to get the 2D quantity with a simple operation.

Reply:
Yes, agreed, better have 1d quantities.